travis/defaults.yml

```diff
@@ -0,0 +1,7 @@
+os: linux
+language: ruby
```

Comment:
out of curiosity, is there anything we might need the system installed ruby for?

Reply:
not really, the travis config parser complains about us not setting any lang, we can also set this as `generic`

Reply:
there's also `language: minimal` (has Docker available) which we used before